dom/src/makeDOMDriver.ts

```diff
@@ -1,75 +1,97 @@
 import {StreamAdapter} from '@cycle/base';
-import {init} from 'snabbdom';
+import {init as initSnabbdom, PatchFunction} from 'snabbdom';
 import xs, {Stream} from 'xstream';
 import {DOMSource} from './DOMSource';
 import {MainDOMSource} from './MainDOMSource';
 import {VNode} from './interfaces';
 import {VNodeWrapper} from './VNodeWrapper';
 import {getElement} from './utils';
 import defaultModules from './modules';
-import {IsolateModule} from './isolateModule';
+import {IsolateModule} from './IsolateModule';
 import {makeTransposeVNode} from './transposition';
 import {EventDelegator} from './EventDelegator';
-import xsSA from '@cycle/xstream-adapter';
+import xsAdapter from '@cycle/xstream-adapter';
 let MapPolyfill: typeof Map = require('es6-map');
 
-function makeDOMDriverInputGuard(modules: any) {
+export {makeDOMDriver}
```

Comment:
We generally put the primary export at the bottom of the file, so we should move this one back. For a moment I was lost looking for the export.